### DISCUSSION TITLE
Custom version of Inter — Advice needed

### DISCUSSION THREAD

**Discussion by type818** [rsms/inter#759] [Q&A] [answered]
I am designing a custom version of [Inter], adding graphic interventions to the letterforms.

I would really appreciate some advice on how to do this best— Is there any way I could get the master glyphs file? If not, which export of the font would be best to re-open in glyphs, to then re-export with my modifications ( OTF/TTF)?

I understand there might be some source code but I have no idea how to use this stuff. Eventually I will contact a much more experienced type designer/engineer than myself to commission this properly, but for now I need to create a working version myself.

Thank you and would appreciate any advice for best practice!

**Comment by rsms** [comment, marked as answer]
1. Source code is in the [`src`](https://github.com/rsms/inter/tree/master/src) directory
2. Read [CONTRIBUTING.md](https://github.com/rsms/inter/blob/master/CONTRIBUTING.md)
3. Make sure to comply with the [license](https://github.com/rsms/inter/blob/master/LICENSE.txt)
4. An experienced type designer knows what to do with the source files

> Is there any way I could get the master glyphs file?

Yes, it's all in the [`src`](https://github.com/rsms/inter/tree/master/src) directory

> I understand there might be some source code but I have no idea how to use this stuff.

You'll have to learn the basics. https://glyphsapp.com/learn is a good resource.
Alt you could just draw in a design program.

**Reply by rsms** [reply]
The author removed the original question (why?), so here is a copy from history:

> I am designing a custom version of [Inter], adding graphic interventions to the letterforms.
>
> I would really appreciate some advice on how to do this best— Is there any way I could get the master glyphs file? If not, which export of the font would be best to re-open in glyphs, to then re-export with my modifications ( OTF/TTF)?
>
> I understand there might be some source code but I have no idea how to use this stuff. Eventually I will contact a much more experienced type designer/engineer than myself to commission this properly, but for now I need to create a working version myself.
>
> Thank you and would appreciate any advice for best practice!

**Reply by type818** [reply]
Hi Rasmus, so sorry — we crossed timing/purposes right to the minute –  as soon as I removed the question you answered (as I had worked out how to get the source files)! My apologies. Thank you very much for responding! 

Could you perhaps advise best practice to add Semi-Bold display as a master in the source glyphs file? There is Thin/Reg/Black as expected. Many thanks

**Reply by rsms** [reply]
No worries :)
